client/src/components/ImageDropZone.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDropzone } from "react-dropzone";
+
+export default function ImageDropZone(props) {
+    const [imgs, setImgs] = React.useState([]);
+    const { getRootProps, getInputProps } = useDropzone({
+        accept: "image/*",
+        multiple: false,
+        onDrop: files => {
+            setImgs(
+                files.map(file =>
+                    Object.assign(file, {
+                        preview: URL.createObjectURL(file)
+                    })
+                )
+            );
+            props.returnImgToParent(files[0]);
```

Comment:
Is only the first file uploaded?

Reply:
I made it so that you can't upload more than one file to a drop zone. It will only accept one file. Adding a pet only requires one photo. For editing the User Profile, I have two drop zones, and I was planning on labeling one for the profile image, and the other for the bg image.  Let me know if you want me to do it differently